sourcecode/apis/contentauthor/composer.json

```diff
@@ -19,7 +19,6 @@
         "cerpus/edlib-resource-kit-laravel": "^0.7.x-dev",
         "cerpus/imageservice-client": "^2.1",
         "cerpus/laravel-rabbitmq-pubsub": "^2.1",
-        "cerpus/questionbank-client": "^2.3.1",
```

Comment:
I think imageservice-client can also be removed now? If I remember correctly, the QB code was the only thing that still used it.

Reply:
Unfortunately it's used in QuestionSet and the Millionaire game